roboflow/models/object_detection.py

```diff
@@ -152,24 +161,23 @@ def predict(
                 img_str = base64.b64encode(buffered.getvalue())
                 img_str = img_str.decode("ascii")
 
-                # Post to API and return response
-                resp = requests.post(
-                    self.api_url,
-                    data=img_str,
-                    headers={"Content-Type": "application/x-www-form-urlencoded"},
-                )
-            else:
+            elif extension == ".jpeg":
                 # Performing inference on a OpenCV2 frame
```

Comment:
This else block seems to have been intended for something else originally; is this breaking functionality? (What type is an OpenCV2 frame?)